modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -132,11 +144,18 @@ def get_pdf_data_wrapper
           { data: {} }
         end
 
-        def process_claim(auto_claim)
-          ClaimsApi::V2::DisabilityCompensationPdfGenerator.perform_async(
-            auto_claim.id,
-            veteran_middle_initial # PDF mapper just needs middle initial
-          )
+        def process_claim(auto_claim, perform_type)
+          if perform_type == 'perform_async'
```

Comment:
@stiehlrod  Wonder if the param might be easier to utilize if it was just a boolean? We could just set the param to `perform_async` as either true or false,  then we could skip the string checking with something like `if perform_async`, thoughts?

Either way works, let me know your thoughts on the above question and I can get this stamped.